adapters/coap/src/main/java/org/eclipse/hono/adapter/coap/option/TimeOption.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.adapter.coap.option;
+
+import org.eclipse.californium.core.coap.Option;
+
+/**
+ * CoAP custom time option.
+ * <p>
+ * Used in CoAP request to indicate that the client wants to get the servers system-time in milliseconds.
+ * Any value in the option as part of a request is ignored.
+ * <p>
+ * If the option is present in a request, the server adds also a time option to the response with the
+ * servers system-time in milliseconds. Also, a client can request this option be included in a response via a
+ * request parameter, "hono-time".
```

Comment:
here you could indeed link to the corresponding property:
> via the {@value #QUERY_PARAMETER_NAME}  request parameter.

Reply:
done